src/models/course.js

```diff
@@ -0,0 +1,61 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class Course extends Model {
+    static associate(models) {
+      Course.hasMany(models.Course, {
+        foreignKey: 'mapsTo',
+        as: 'mapsFromCourse',
+      });
+      Course.belongsTo(models.Course, {
+        foreignKey: 'mapsTo',
+        as: 'mapsToCourse',
+      });
+    }
+  }
+  Course.init({
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+      allowNull: false,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
```

Comment:
String is a [VARCHAR(255)](https://sequelize.org/docs/v7/models/data-types/) Is that going to be enough or should it be text?

Reply:
Updted to text.